src/com/jalasoft/search_files/utils/Search_Validation.py

```diff
@@ -0,0 +1,54 @@
+import re
+import bitmath
+
+
+class SearchValidation(object):
+    def __init__(self):
+        pass
+
+    def string1_exists_within_string2(self, string_1, string_2, flag):
+        case_sensitive = bool(flag)
+        if case_sensitive is True:
+            search_match = re.search(string_1, string_2, re.IGNORECASE)
+            if search_match is None:
+                return False
+            else:
+                return True
+        else:
+            search_match = re.search(string_1, string_2)
+            if search_match is None:
+                return False
+            else:
+                return True
+
+    def string1_equal_to_string2(self, string_1, string_2, flag):
```

Comment:
- "string1_equal_to_string2"  and "string1_exists_within_string2" functions call to " re.search(string_1, string_2, re.IGNORECASE)"  when case_sensitive is True.
- Both functions have similar body and inputs and different function names. 
- Please, update those function,  IGNORECASE means that A = a